payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -35,12 +54,58 @@ class GooglePayPaymentMethodLauncher @AssistedInject internal constructor(
     @Assisted private val config: Config,
     @Assisted private val readyCallback: ReadyCallback,
     @Assisted private val activityResultLauncher: ActivityResultLauncher<GooglePayPaymentMethodLauncherContract.Args>,
+    @Assisted private val skipReadyCheck: Boolean,
+    private val context: Context,
     private val googlePayRepositoryFactory: (GooglePayEnvironment) -> GooglePayRepository,
-    analyticsRequestFactory: AnalyticsRequestFactory,
-    analyticsRequestExecutor: AnalyticsRequestExecutor
+    @Named(PRODUCT_USAGE) private val productUsage: Set<String>,
+    @Named(PUBLISHABLE_KEY) private val publishableKeyProvider: () -> String,
+    @Named(STRIPE_ACCOUNT_ID) private val stripeAccountIdProvider: () -> String?,
+    @Named(ENABLE_LOGGING) private val enableLogging: Boolean = BuildConfig.DEBUG,
```

Comment:
Passing default values to a Injectable constructor is a bit confusing, I understand that this constructor will be called manually when used standalone and the default values are used in those cases, could we add some comments here to explain that?

Reply:
Done.